pkg/cmd/server/server.go

```diff
@@ -459,6 +459,15 @@ func (c *Config) Complete(ctx context.Context) (RunnableServer, error) {
 	closeables.AddCloser(gatewayCloser)
 	closeables.AddWithoutError(gatewayServer.Close)
 
+	infoCollector, err := telemetry.SpiceDBClusterInfoCollector(context.TODO(), "environment", c.DatastoreConfig.Engine, ds)
```

Comment:
please use the context
```suggestion
	infoCollector, err := telemetry.SpiceDBClusterInfoCollector(ctx, "environment", c.DatastoreConfig.Engine, ds)
```

Reply:
thanks for this, i did some refactoring